### PR TITLE
add test for ledger entries

### PR DESCRIPTION
**Pull request by daro-maj**

### CODE REVIEW

File: tests/portfolio/test_ledger_entries.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+@pytest.fixture(scope="module")
+def vega(request):
+    with init_vega(request) as vega:
+        yield vega
+
+
+@pytest.fixture(scope="module")
+def continuous_market(vega):
+    return setup_continuous_market(vega)
```
Comment (bwallacee) [inline tests/portfolio/test_ledger_entries.py:17]: I don't think we need a market created for this test?

We could just use vega, like in the test_navigation.

File: tests/portfolio/test_ledger_entries.py [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+
+@pytest.mark.usefixtures("page", "auth", "risk_accepted")
+def test_ledger_entries_downloads(continuous_market, vega: VegaService, page: Page):
```
Comment (bwallacee) [inline tests/portfolio/test_ledger_entries.py:21]:
```suggestion
def test_ledger_entries_downloads(continuous_market,  page: Page):
```

It doesn't look like we are using vega in the test.

File: tests/portfolio/test_ledger_entries.py [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+        page.get_by_test_id("ledger-download-button").click()
+    download = download_info.value
+    # Wait for the download process to complete and save the downloaded file in the Downloads directory
+    download.save_as(os.path.join(downloads_directory, download.suggested_filename))
+
+    # Verify the download by asserting that the file exists
+    downloaded_file_path = os.path.join(downloads_directory, download.suggested_filename)
+    assert os.path.exists(downloaded_file_path), f"Download failed! File not found at: {downloaded_file_path}"
+    print(downloads_directory)
+    print("Download successful!")
```
Comment (bwallacee) [inline tests/portfolio/test_ledger_entries.py:42]:
```suggestion
```
Should we remove these prints?

File: tests/portfolio/test_ledger_entries.py [hunk 4]
```diff
@@ -0,0 +1,43 @@
+import os
+import pytest
+from playwright.sync_api import Page, expect
+from vega_sim.service import VegaService
```
Comment (bwallacee) [inline tests/portfolio/test_ledger_entries.py:4]:
```suggestion
```

**Review by bwallacee — CHANGES_REQUESTED** [review]

File: tests/portfolio/test_ledger_entries.py (hunk 1 above)
Comment (daro-maj) [reply tests/portfolio/test_ledger_entries.py]: done

File: tests/portfolio/test_ledger_entries.py [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+@pytest.mark.usefixtures("page", "auth", "risk_accepted", "continuous_market")
+def test_ledger_entries_downloads(page: Page):
+    page.goto("/#/portfolio")
+    page.get_by_test_id("Ledger entries").click()
+    expect(page.get_by_test_id("ledger-download-button")).to_be_enabled()
+
+    # 7007-LEEN-001
+    # Get the user's Downloads directory
+    downloads_directory = os.path.expanduser("~") + "/Downloads/"
+    page.pause()
```
Comment (bwallacee) [inline tests/portfolio/test_ledger_entries.py:14]:
```suggestion
```

**Review by bwallacee — APPROVED** [review]

**Pull request merged**